specification/metrics/listunitprice.md

```diff
@@ -1,8 +1,8 @@
 # List Unit Price
 
-The List Unit Price represents the suggested provider published unit price for a single pricing measurement unit, which incorporates volume/tier-based rates while excluding any negotiated or commitment-based discounts. This price is denominated in the [Billing Currency](#billingcurrency). It is used to calculate List Cost and savings based on rate optimization activities.
+The List Unit Price represents a suggested provider-published unit price for a single pricing measurement unit (identified by a [Sku Price Id](#skupriceid)), which incorporates free-tier, volume/tier-based, BYOL-based and dynamically priced SKU rates, while excluding any other discounts or the amortization of upfront charges (one-time or recurring). This price is denominated in the [Billing Currency](#billingcurrency). The List Unit Price can be used for calculating savings based on rate optimization activities. In this context, activities such as increased consumption leading to lower volume/tier-based rates, the use of pre-owned software licenses (BYOL - Bring Your Own License), leveraging interruptible resources and/or services, or optimizing usage to take advantage of dynamic pricing models are not considered rate optimization activities.
```

Comment:
>   a suggested provider-published unit price for a single pricing measurement unit (identified by a [Sku Price Id](#skupriceid))

I am not clear why the single pricing measurement unit is identified by the SKU Price ID. My understanding is that a SKU may have SKU Prices,. which are all identified with different SKU Price IDs:
- List Price (what we are discussing here) = public on-demand price (no commitment)
- Negotiated Price = negotiated on-demand price (no commitment)
- Effective Price  = Negotiated price with commitment-based discount

So I would propose:

>   a suggested provider-published unit price for *the* single pricing measurement unit *that is billed in the charge*

Reply:
Or maybe the "(identified by a [Sku Price Id])" means there must be a SKU Price ID for the list price?
If yes, it may be confusing, the SKU Price ID of the list price may be different from the SKU Price ID of the record.

To make it clearer, maybe:

> If list price applies from the charge, then the SKU Price ID of the charge must identify a provider-published unit price

Reply:
One SKU ID can have multiple SKU Price IDs, multiple ListUnitPrices in case of volume/tiered-based rating. 
E.g.:

- tier 1: 0 - 1000 requests; 1.00 $
- tier 2: 1001 - 10000 requests; 0.90$
- etc.